### DISCUSSION TITLE
When a user is part of a team, give access to /access/$project/#teams

### DISCUSSION THREAD

**Discussion by emmapeel2** [WeblateOrg/weblate#11741] [Ideas] [answered]
I have created groups for reviewers, one group per language.

I have given some people permission to admin their groups, so they can add more reviewers as they see fit, and self-manage their team.

I also have given permissions to the Localization Lab, that supports us to translate our project and gets us reviewers between the many things they do for us (I love you guys!).

I think when you are administering a group, and even if you are just part of one, you should be able to see on  /access/$project/#teams a list of the groups you are part of, with some information.

## How to reproduce

- invite a new user to a team
- logged in as that user, try to reach  /access/$project/#teams

## What happens now

You get a nasty message that makes you feel an outsider (ok im exaggerating a little)

```
Permission Denied
Insufficient privileges to display this page. 

```

## what should happen instead

You get a list of your current teams, and if not, some helpful message about 'how to give feedback in weblate' or information about the project.

**Comment by nijel** [comment]
You can see your teams at `/accounts/profile/#teams`.

**Comment by emmapeel2** [comment]
ey @nijel: The view at /accounts/profile/#teams could be improved by showing the weblate-projects in the 'Projects' column. I am admin in several projects and the view does not help, see:


<img width="1872" height="919" alt="slika" src="https://github.com/user-attachments/assets/dd0f78ba-d3a7-488e-a0a6-b10f152d0b11" />

Thanks!

**Reply by nijel** [reply, marked as answer]
It is actually supposed to be there, but some optimization broke that. https://github.com/WeblateOrg/weblate/pull/15821 should fix it.
